cmd/cli/serve/serve.go

```diff
@@ -260,7 +260,7 @@ func serve(cmd *cobra.Command, cfg types.Bacalhau, fsRepo *repo.FsRepo) error {
 
 	if !cfg.DisableAnalytics {
 		err = analytics.SetupAnalyticsProvider(ctx,
-			analytics.WithNodeNodeID(sysmeta.NodeName),
+			analytics.WithNodeID(sysmeta.NodeName),
```

Comment:
@wdbaruni is collecting the name of the node acceptable, just want to double check.